openlayers/openlayers.d.ts

```diff
@@ -12619,7 +12619,7 @@ declare module olx {
          *     visible: (boolean|undefined)}}
          */
         interface VectorOptions {
-            renderOrder: (feature1: ol.Feature, feature2: ol.Feature) => number;
+            renderOrder?: (feature1: ol.Feature, feature2: ol.Feature) => number;
```

Comment:
I have been using OpenLayers for 2 years and wasn't even aware of the renderOrder option until yesterday.  Let's go ahead with this change.  The openlayers code obviously handles an undefined renderOrder and I think it would be preferable that the type declarations not flag the absence of this option as an error.